config/routes.rb

```diff
@@ -23,6 +23,10 @@
 
   post "global_search" => 'global_search#index', as: "global_search"
 
+  resources :users, only: [] do
```

Comment:
I would expect that we already have a `users` route. Am I wrong?

Reply:
We do, but it's nested under facilities, whereas we want user preferences to exist regardless of facilities.